geom/alg_equals_exact.go

```diff
@@ -18,17 +18,29 @@ func newEqualsExactOptionSet(opts []EqualsExactOption) equalsExactOptionSet {
 }
 
 // Tolerance modifies the behaviour of the EqualsExact method by allowing two
-// geometry control points be be considered equal if they are within the given
-// euclidean distance of each other.
+// geometry control points be be considered equal if their XY coordinates are
+// within the given euclidean distance of each other.
```

Comment:
Nit: Perhaps this function should be called `XYTolerance` to make clear this is a specialised distance metric involved. Coming from a 3D background, I would find it strange that Z is not included, so making that clear in the function's name could help avoid invalid assumptions.

Reply:
Agreed, renamed (we'll also want to add tolerance functions for Z and M, I'll created tickets for those).